package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "service-template-node",
-  "version": "0.0.0",
+  "version": "0.1.0",
```

Comment:
My interpretation of http://semver.org/ is that the first release number should be 0.1.0

"How should I deal with revisions in the 0.y.z initial development phase?

The simplest thing to do is start your initial development release at 0.1.0 and then increment the minor version for each subsequent release."